src/Plutip/Server.purs

```diff
@@ -184,12 +179,31 @@ stopOgmiosDatumCache = liftEffect <<< kill SIGINT
 
 mkClusterContractCfg
   :: forall (r :: Row Type)
-   . ContractConfig r
+   . PlutipConfig
   -> ClusterStartupParameters
-  -> ContractConfig r
-mkClusterContractCfg contractCfg clusterParams =
-  flip (over ContractConfig) contractCfg
-    \x -> x
+  -> Aff (ContractConfig ())
+mkClusterContractCfg plutipCfg _clusterParams = do
+  ogmiosWs <- QueryM.mkOgmiosWebSocketAff plutipCfg.logLevel
+    QueryM.defaultOgmiosWsConfig
+      { port = plutipCfg.ogmiosConfig.port
+      , host = "localhost"
+      }
+  datumCacheWs <-
+    QueryM.mkDatumCacheWebSocketAff plutipCfg.logLevel
+      QueryM.defaultDatumCacheWsConfig
+        { port = plutipCfg.ogmiosDatumCacheConfig.port
+        , host = "localhost"
+        }
+  usedTxOuts <- newUsedTxOuts
+  pure $ ContractConfig
+    { ogmiosWs
+    , datumCacheWs
+    , wallet: Nothing -- TODO: use KeyWallet when it's ready
```

Comment:
We can use the key wallet now, no?

Reply:
Yes, but it's out of scope of this PR. Note that we are going to merge this PR into a branch that is itself a draft.